### PR TITLE
[REG-19] Update the registries navbar

### PR DESCRIPTION
**Pull request by chrisseto**
## Purpose
Implement the new registries navbar.

## Summary of Changes
  * Adds the following components
    * nav-bar
    * icon-input
    * registries-navbar
    * x-dummy

## Feature Flags
* Hidden behind the feature flag `ember_registries_new_style`

## QA Notes
* Styling is more or less custom so there might be some edge cases on other browsers

## Ticket

https://openscience.atlassian.net/browse/REG-19

# Reviewer Checklist

- [ ] meets requirements
- [ ] easy to understand
- [ ] DRY
- [ ] testable and includes test(s)
- [ ] changes described in `CHANGELOG.md`

### CODE REVIEW

File: lib/registries/addon/components/nav-bar/template.hbs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<nav data-test-nav role="navigation" class="{{this.media.classNames}}" local-class="Nav {{if this.dark 'Dark' 'Light'}}">
+    {{yield (hash
```
[the remark this answers is not in the document]
Comment (chrisseto) [reply lib/registries/addon/components/nav-bar/template.hbs]: `nav-bar` is also used for the mobile navbar. I wanted to keep the styling as consistent as I could for the two!

File: lib/registries/addon/components/registries-navbar/template.hbs [hunk 2]
```diff
@@ -0,0 +1,110 @@
+{{#nav-bar dark=true as |nav|}}
+    {{!-- Right Side (Band, Service Dropdown) --}}
```
[the remark this answers is not in the document]
Comment (chrisseto) [reply lib/registries/addon/components/registries-navbar/template.hbs]: Oh 🦆, I think I did.

**Comment by chrisseto** [comment]
Contained by https://github.com/CenterForOpenScience/ember-osf-web/pull/430
